config/jobs/kubernetes-sigs/cluster-api-provider-gcp/cluster-api-provider-gcp-presubmits-main.yaml

```diff
@@ -300,3 +300,31 @@ presubmits:
     annotations:
       testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-gcp
       testgrid-tab-name: pr-apidiff-main
+  - name: pull-cluster-api-provider-gcp-coverage
+    always_run: true
```

Comment:
`always_run` false please. once it goes green a follow up PR we can set it to `true`

Reply:
Done